commands/build-image.ts

```diff
@@ -95,20 +93,28 @@ export async function buildImage(actionContext: IActionContext, dockerFileUri?:
     if (defaultContextPath && defaultContextPath !== '') {
         contextPath = defaultContextPath;
     }
+    let absFilePath: string = path.join(vscode.workspace.workspaceFolders[0].uri.fsPath, dockerFileItem.relativeFolderPath, dockerFileItem.relativeFilePath);
```

Comment:
Paths are relative to rootFolder

Reply:
It's relativeFile*Path*, not relativeFile*Name* (which wouldn't make sense).  So I expect you don't need relativeFolderPath